app/Repositories/PostRepository.php

```diff
@@ -73,7 +73,7 @@ public function create(array $data, $signupId)
             $fileUrl = 'default';
         }
 
-        $signup = Signup::find($signupId);
+        $signup = Signup::withCount('posts')->find($signupId);
```

Comment:
@sbsmith86 just curious why we need to query for this `withCount`?

Reply:
ahh yep. This allows me to then check if this is not the first post under a signup because it adds a `post_count` property to the signup (see line 101). Depending on whether or not this is the first post or not, we handle edge cases differently. For example, if it is not the first post then when the user submits a quantity that is equal to what we already have, we store it as `0`

Reply:
ahhh got it - thanks for clarifying @sbsmith86 !